src/components/EventList.spec.tsx

```diff
@@ -96,15 +92,15 @@ describe('EventList', () => {
             });
     });
 
-    it('contains a signup button when user is not logged in', () => {
+    it('contains an account signup button when user is not logged in', () => {
```

Comment:
Why the change to this label? Has the purpose or functionality of the test changed since Neha implemented it? The button is (and should be) a regular sign-up button, but instead of signing up, it triggers a modal dialog asking the user to register an account before they proceed. The new label implies to me that the registration (account signup) button is visible in the `EventList`, which I believe it's not, at least not conceptually.

Reply:
Will revert this to it's original state since it doesn't test the modal as I thought.